src/api/queries/get_user.ts

```diff
@@ -18,7 +19,7 @@ export const useGetUserQuery = (username: string | undefined) => {
   return useQuery<User | null>({
     queryKey: ["get-user", username],
     queryFn: () => getUser(username!),
-    enabled: username != null,
+    enabled: !!username,
```

Comment:
I prefer an explicit check, not using `!!`. While it may syntactically make sense, explicitly checking helps to avoid having to double think what the boolean condition does. For instance, are we disallowing empty strings or not?